libraries/drivers_db_sqlite.rb

```diff
@@ -7,6 +7,21 @@ class Sqlite < Base
       allowed_engines :sqlite, :sqlite3
       packages debian: 'libsqlite3-dev', rhel: 'sqlite-devel'
 
+      def deploy_before_symlink
+        link_sqlite_database
+      end
+
+      def validate_node_engine
+        unless node_engine
+          framework = Drivers::Framework::Factory.build(context, app)
+          deploy = context.node['deploy']
+          deploy[app['shortname']][driver_type] ||= {}
+          deploy[app['shortname']][driver_type][:database] = "db/#{app['shortname']}_#{framework.deploy_env}.sqlite"
+          context.node['deploy'] = deploy
+        end
+        :node_engine
```

Comment:
Please use guard clause here, so instead this big unless:

```ruby
return :node_engine if node_engine
framework = ....
```

Reply:
The only thing is, the `return :node_engine` happens in _all_ cases, so it's not a guard clause per se. But I'll clean it up to make it more readable.